langchain/src/chains/base.ts

```diff
@@ -119,10 +130,10 @@ export abstract class BaseChain extends BaseLangChain implements ChainInputs {
    */
   async apply(
     inputs: ChainValues[],
-    callbackManagers?: CallbackManager[]
+    callbacks?: CallbackManager[] | BaseCallbackHandler[][]
```

Comment:
Oh I hadn't realised before that here we accept an array of callback managers. I can see the usefulness, but someone can accidentally pass an array of handlers here (if not using typescript)?

Reply:
Yeah tbh, I'm not sure how much this method is used rn. We can restrict it to one callback manager or a single array of handlers?